tests/tracking/test_tracking.py

```diff
@@ -538,6 +539,53 @@ def test_log_dict(subdir, extension):
             assert loaded == dictionary
 
 
+@pytest.mark.skipif(
+    importlib.util.find_spec("matplotlib") is None, reason="matplotlib is not installed"
+)
```

Comment:
Should we use `@pytest.mark.large` instead?

Reply:
Seems fine to mark the test as large IMO, rather than risk skipping it if we fail to install matplotlib before running tests